core/common/scale/util.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "util.hpp"
+
+#include <boost/predef.h>
```

Comment:
```suggestion

```

Reply:
It is about include order, right? What do you propose?